src/main/java/com/iexec/sms/secret/app/TeeTaskRuntimeSecret.java

```diff
@@ -27,17 +27,23 @@
 @EqualsAndHashCode(callSuper = true)
 @NoArgsConstructor
 @Entity
-public class AppRuntimeSecret extends Secret {
+public class TeeTaskRuntimeSecret extends Secret {
```

Comment:
I don't think it's still relevant to extend from `Secret`. Former `address` field doesn't make sense by itself now. I would move it in `TeeTaskRuntimeSecret` as `deployedObjectAddress`.
We might update encryption/decryption feature for this class/related-service

Reply:
Alright, I agree with you. That's cleaner this way.